testapps/iOSImageFormatProfilingTestApp/test.xcodeproj/xcuserdata/chenyang.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Most of these files shouldn't be checked in. Can we update the `.gitignore` file with Xcode/Apple defaults (https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore seems like a good starting point?) and make sure we don't check in things like this local breakpoint list?